CHANGELOG.md

```diff
@@ -43,12 +73,53 @@ certainly be made before the closing of the merging window.
   now deprecated. This release is backwards-compatible with the previous model
   and all of your currently defined APIs and matching rules are still
   supported.
-
-Additionally, this release will also include all the changes listed under the
-[0.12.2](#0122) release.
+  [39d9e5e](https://github.com/Kong/kong/commit/39d9e5e5224f1af0b89d108d4bf28281836d893e)
+
+- :fireworks: New Data Access Object layer (DAO). The new layer is available
+  at the `kong.db` namespace instead of `kong.dao`. It offers many advantages:
+      - Simplified interface
+      - Automatic method generation
+      - Improved error description and handling
+      - Standardization of input and output parameters
+      - Automatic pagination
+  `kong.db` is used only in Routes and Services for now. It will gradually
+  replace the old `kong.dao` layer.
+  [f771ed5](https://github.com/Kong/kong/commit/f771ed5c7a64390c9f8b3ef06bbddabc24dd9334)
```

Comment:
reminder that this change is considered private until the interface is publicly available, pluggable, and documented